atari-arcade-cabinets/scene.json

```diff
@@ -0,0 +1,56 @@
+{
+  "ecs7": true,
+  "runtimeVersion": "7",
+  "display": {
+    "title": "Atari Arcade Cabinets",
+    "description": "Atari Arcade Cabinets",
+    "navmapThumbnail": "images/scene-thumbnail.png",
+    "favicon": "favicon_asset"
+  },
+  "owner": "",
+  "contact": {
+    "name": "Seba",
+    "email": ""
+  },
+  "main": "bin/index.js",
+  "tags": [],
+  "scene": {
+    "parcels": [
+      "74,10",
+      "75,10",
+      "74,11",
+      "75,11"
+    ],
+    "base": "74,10"
+  },
```

Comment:
These coords are colliding with other scene. Please select proper coords and run `npm run update-parcels` before pushing again

Reply:
my bad, I forgot run update-parcels